src/widgets/DataTable/Row.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable react/forbid-prop-types */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { View, StyleSheet } from 'react-native';
+
+const Row = ({ rowData, rowState, rowKey, renderCells }) => {
+  console.log('=================================');
```

Comment:
Should we get rid of all the logs in the components or did you want to keep them?

Reply:
Nah, I think keep them until we're happy with our implementation of customerInvoicePage